release/ray_release/tests/test_step.py

```diff
@@ -0,0 +1,30 @@
+import sys
+
+import pytest
+
+from ray_release.bazel import bazel_runfile
+from ray_release.buildkite.step import get_step
+from ray_release.configs.global_config import init_global_config
+from ray_release.test import Test
+
+init_global_config(bazel_runfile("release/ray_release/configs/oss_config.yaml"))
+
+
+def _stub_test(val: dict) -> Test:
```

Comment:
could you add a comment saying that this is a fake test stub that represents a release test? or something?

so that it is clear that this is not testing a thing called "stub".